### PR TITLE
Merging Master to mainline.

### PR DESCRIPTION
**Pull request by charlypa**
Create a pull request on initial commit

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
+-abcoll
```
Comment (abose) [inline requirements.txt:1]: What is this file used for?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[metadata]
+# replace with your username:
+name = crawl-appinfo-pkg-charly
+version = 0.0.2
+author = charly abraham
```
Comment (abose) [inline setup.cfg:5]: and core.ai

File: README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Introduction
+App info extractor is a utility to extract and analyze customer reviews and critical metrics of apps from multiple sources like google play store, Apple play store, and any other sources.This utility is highly configurable to extract and give relevant information about any app in a meaningful way.
+
+## How it works?
+This utility takes config.yaml as input. Configuration for each app is self-contained in configs. Once a configuration is specified in config.yaml, the utility will iteratively go through each app, crawl the relevant information, and store the data locally as a CSV file.
```
Comment (abose) [inline README.md:5]: Would be  helpful to give sample usage CLI 
`>command name agr1 arg2`

**Review by abose — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (charlypa) [reply requirements.txt:1]: @abose this is auto-generated using dephell
https://dephell.readthedocs.io/installation.html

File: setup.cfg (hunk 2 above)
Comment (charlypa) [reply setup.cfg:5]: will change it

File: requirements.txt (hunk 1 above)
Comment (abose) [reply requirements.txt:1]: What is dephell used for.

File: README.md (hunk 3 above)
Comment (charlypa) [reply README.md:5]: done

File: requirements.txt (hunk 1 above)
Comment (charlypa) [reply requirements.txt:1]: This is used to lock the package versions in my current workspace. As I was experimenting with it earlier, I did not create virtualenv and requirement.txt at the start. In order to ensure the smooth working of this package in your system, we need to lock all the package versions which were working for me. Dephell helped me to do it. 

Another alternative would be  ``` pip freeze ``` .  however, I used dephell

**Review by abose — APPROVED** [review]

**Pull request merged**